src/extensions/replay/sessionrecording.ts

```diff
@@ -807,6 +834,11 @@ export class SessionRecording {
         this._tryAddCustomEvent('$posthog_config', {
             config: this.instance.config,
         })
+
+        logger.info(LOGGER_PREFIX + ' started', {
```

Comment:
moved the log here since we were logging started twice when we weren't really just the previous location is idempotent and called multiple times